tutorials/install-distro.md

```diff
@@ -0,0 +1,72 @@
+# Distribution Specific Installation Instructions
+
+## Supported Versions
+Below is a compatiblity matrix between versions of CRI-O (y-axis) and distributions (x-axis)
+
+|      | Fedora 31+ | openSUSE | CentOS_8 | CentOS_8_Stream | CentOS_7 | Debian_Unstable | Debian_Testing | Debian 10 | Rasbian_10 | xUbuntu_20.04 | xUbuntu_19.10 | xUbuntu_19.04 | xUbuntu_18.04 |
+| ---- | ---------- | -------- | -------- | --------------- | -------- | --------------- | -------------- | --------- | ---------- | ------------- | ------------- | ------------- | ------------- |
+| 1.18 | x          | x        | x        | x               |          | x               | x              |           |            | x             |               |               |               |
+| 1.17 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+| 1.16 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+
+To install, choose a supported version for your operating system, and export it as a variable, like so:
+`export VERSION=1.18`
+
+
+## Installation Instructions
+
+### Fedora 31 or later
+```shell
+sudo dnf module enable cri-o:$VERSION
+sudo dnf install cri-o
+```
+
+### openSUSE:
+```shell
+sudo zypper install cri-o
+```
+
+
+### Other yum based operating systems
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Centos 8         | `CentOS_8`        |
+| Centos 8 Stream  | `CentOS_8_Stream` |
+| Centos 7         | `CentOS_7`        |
+
+
+And then run the following as root:
+```shell
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
+yum install cri-o
+```
+
+
+### Deb based operating systems
```

Comment:
```suggestion
### Debian based operating systems
```

Reply:
Might as well call it "apt based", if you want it to rhyme with "yum based"

Reply:
I went with apt based